### ISSUE TITLE
Should we add a requirements.txt file?

### ISSUE THREAD

**Issue by ibnesayeed** [mementoweb/timegate#6]
In the Getting Started guide, following command is given to install the dependencies:

```
$ echo 'uWSGI>=2.0.3 ConfigParser>=3.3.0r2 python-dateutil>=2.1 requests>=2.2.1 werkzeug>=0.9.6 lxml>=3.4.1' | xargs pip install
```

We can add a requirements.txt file as a common way to install dependencies using pip. This way it will be easier to update dependenices as code changes, and people wont have to refer to a Wiki page to know about the dependencies.

**Comment by tiborsimko** [comment]
Note that this is solved in PR #5 via `setup.py` so that `pip` installs all the necessary dependencies automatically.

**Comment by jirikuncar** [comment]
IMHO it's duplicate of #3
